src/org/zaproxy/zap/extension/replacer/ReplacerParam.java

```diff
@@ -0,0 +1,230 @@
+        final String defaultListArray[][] = {
+                { 
+                        "Remove CSP", 
+                        ReplacerParamRule.MatchType.resp_header.name(), 
+                        "Content-Security-Policy", 
+                        "",
+                        "false", 
+                        "", 
+                        "false" },
+                {
+                        "Remove HSTS",
+                        ReplacerParamRule.MatchType.resp_header.name(),
+                        "Strict-Transport-Security",
+                        "",
+                        "false",
+                        "",
+                        "false" },
+                {
+                        "Replace User-Agent with shellshock attack",
+                        ReplacerParamRule.MatchType.req_header.name(),
+                        "User-Agent",
+                        "() {:;}; /bin/cat /etc/passwd",
+                        "false",
+                        "",
+                        "false" } };
+
+        for (String row[] : defaultListArray) {
+            boolean regex = row[3].equalsIgnoreCase("true") ? true : false;
+            boolean enabled = row[5].equalsIgnoreCase("true") ? true : false;
+            defaultList.add(new ReplacerParamRule(row[0], MatchType.valueOf(row[1]), row[2], regex, row[3], null, enabled));
+        }
+    }
+
+    public ReplacerParam() {
+        super();
+        setDefaultList();
+    }
+
+    @Override
+    protected void parse() {
+        try {
+            List<HierarchicalConfiguration> fields = ((HierarchicalConfiguration) getConfig()).configurationsAt(ALL_RULES_KEY);
+            this.rules = new ArrayList<>(fields.size());
+            enabledRuleNames = new ArrayList<>(fields.size());
+            List<String> tempTokensNames = new ArrayList<>(fields.size());
+            for (HierarchicalConfiguration sub : fields) {
+                String desc = sub.getString(RULE_DESCRIPTION_KEY, "");
+                if (!"".equals(desc) && !tempTokensNames.contains(desc)) {
+                    boolean enabled = sub.getBoolean(RULE_ENABLED_KEY, true);
+                    boolean regex = sub.getBoolean(RULE_REGEX_KEY, true);
+                    String matchStr = sub.getString(RULE_MATCH_STRING_KEY, "");
+                    MatchType matchType = MatchType.valueOf(sub.getString(RULE_MATCH_TYPE_KEY, MatchType.resp_body_str.name()));
+                    String replace = sub.getString(RULE_REPLACEMENT_KEY, "");
+                    String initStr = sub.getString(RULE_INITIATORS_KEY, "");
+                    List<Integer> initList = null;
+                    if (!StringUtils.isEmpty(initStr)) {
+                        initList = new ArrayList<Integer>();
+                        String[] initStrArray = initStr.replace("[", "").replace("]", "").split(",");
+                        for (String str : initStrArray) {
+                            initList.add(Integer.parseInt(str.trim()));

```

Comment:
Should handle `NumberFormatException`? It's possible that the array contains an empty string (which would fail to parse as int), see following comment.